core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -172,7 +172,7 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
   }
 
   /** Length of time to wait while draining listener events. */
-  val WAIT_TIMEOUT_MILLIS = 10000
+  val WAIT_TIMEOUT_MILLIS = 30000
```

Comment:
I was seeing some intermittent timeouts on this test so increased, I can back it out if people think this is to long.